lib/idnow/sftp_client.rb

```diff
@@ -2,15 +2,18 @@
 
 module Idnow
   class SftpClient
-    def initialize(host:, username:, password:)
+    def initialize(host:, username:, password:, timeout: nil)
       @host = URI.parse(host).host
       @username = username
       @password = password
+      @timeout = timeout
     end
 
     def download(file_name)
       data = nil
-      Net::SFTP.start(@host, @username, password: @password) do |sftp|
+      options = { password: @password }
```

Comment:
Why not `options = { host: @host, username: @username, password: @password }` like you did in `client.rb` and then just `Net::SFTP.start(options)`?

Reply:
I tried to keep in line with the signature of `Net::SFTP.start`:
```ruby
def self.start(host, user, options={}, &block)
```
(see: https://github.com/net-ssh/net-sftp/blob/9d4e24ad006f309efd88e518b28b2cfe292a6d1a/lib/net/sftp.rb#L30)